lib/apiservers/engine/backends/middleware/middleware.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package middleware
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	vicdns "github.com/vmware/vic/lib/dns"
+
+	context "golang.org/x/net/context"
```

Comment:
Not sure which line you're commenting on here @hickeng

Reply:
context is part of the core go libraries now and no longer need to pull in golang.org/x/net/context.  We still pull that in in parts of the persona because Docker code that we vendor still uses that package.  Although the library is basically the same, they're namespaced.

Reply:
This appears to be necessary not because mdns is out of date but because the Docker interface that I'm implementing still uses `golang.org/x/net/context` so in order to match the interface definition this definition of context must be used.